code/modules/research/xenobiology/xenobio_camera.dm

```diff
@@ -220,11 +245,13 @@
 	if(GLOB.cameranet.checkTurfVis(remote_eye.loc))
 		for(var/mob/living/carbon/monkey/M in remote_eye.loc)
 			if(M.stat)
-				M.visible_message("[M] vanishes as [M.p_theyre()] reclaimed for recycling!")
+				M.visible_message("[M] vanishes as [p_theyre()] reclaimed for recycling!")
```

Comment:
Should this use `[M.p_theyre()]` or `[p_theyre()]` for the pronoun embed? What's the difference?

Reply:
It should 100% be M.p_theyre(), the difference is that the former refers to the monkey and the latter refers to nothing at all and will runtime error